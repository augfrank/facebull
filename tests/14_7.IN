M0   C0   C3  2793
M1   C0   C4  2690
M2   C0   C5  2071
M3   C0   C6  2383
M4   C0   C7  2846
M5   C0   C9  2964
M6   C0   C10  2249
M7   C0   C11  2251
M8   C0   C13  2516
M9   C1   C0  2892
M10   C1   C2  2460
M11   C1   C3  2513
M12   C1   C4  2987
M13   C1   C5  2261
M14   C1   C6  2908
M15   C1   C7  2200
M16   C1   C8  2055
M17   C1   C11  2714
M18   C1   C12  2893
M19   C1   C13  2151
M20   C2   C0  2817
M21   C2   C4  2001
M22   C2   C5  2035
M23   C2   C6  2107
M24   C2   C8  2283
M25   C2   C9  2145
M26   C2   C10  2803
M27   C2   C11  2945
M28   C2   C12  2259
M29   C2   C13  2125
M30   C3   C0  2615
M31   C3   C1  2293
M32   C3   C2  2395
M33   C3   C4  2439
M34   C3   C5  2250
M35   C3   C6  2666
M36   C3   C7  2098
M37   C3   C8  2824
M38   C3   C9  2382
M39   C3   C10  2941
M40   C3   C11  2050
M41   C3   C12  2578
M42   C3   C13  2149
M43   C4   C0  2155
M44   C4   C1  2311
M45   C4   C2  2130
M46   C4   C3  2719
M47   C4   C5  2725
M48   C4   C6  2895
M49   C4   C9  2465
M50   C4   C13  2457
M51   C5   C0  2647
M52   C5   C1  2552
M53   C5   C2  2193
M54   C5   C3  2680
M55   C5   C4  2586
M56   C5   C6  2879
M57   C5   C8  2697
M58   C5   C9  2065
M59   C5   C10  2018
M60   C5   C11  2005
M61   C5   C12  2009
M62   C5   C13  2299
M63   C6   C0  2408
M64   C6   C1  2367
M65   C6   C2  2964
M66   C6   C3  2758
M67   C6   C4  2076
M68   C6   C5  2937
M69   C6   C7  2288
M70   C6   C8  2736
M71   C6   C9  2750
M72   C6   C10  2977
M73   C6   C11  2384
M74   C6   C12  2088
M75   C6   C13  2736
M76   C7   C0  2868
M77   C7   C2  2016
M78   C7   C3  2963
M79   C7   C4  2529
M80   C7   C5  2232
M81   C7   C6  2163
M82   C7   C8  2801
M83   C7   C9  2864
M84   C7   C10  2735
M85   C7   C12  2444
M86   C7   C13  2379
M87   C8   C0  2414
M88   C8   C1  2967
M89   C8   C2  2881
M90   C8   C3  2088
M91   C8   C4  2017
M92   C8   C5  2816
M93   C8   C7  2994
M94   C8   C9  2621
M95   C8   C10  2362
M96   C8   C11  2187
M97   C8   C12  2372
M98   C8   C13  2895
M99   C9   C0  2596
M100   C9   C1  2196
M101   C9   C2  2228
M102   C9   C3  2235
M103   C9   C4  2730
M104   C9   C5  2621
M105   C9   C7  2565
M106   C9   C10  2787
M107   C9   C11  2173
M108   C9   C12  2733
M109   C10   C2  2523
M110   C10   C3  2254
M111   C10   C4  2100
M112   C10   C9  2984
M113   C10   C11  2312
M114   C10   C12  2595
M115   C10   C13  2347
M116   C11   C0  2337
M117   C11   C1  2946
M118   C11   C2  2277
M119   C11   C4  2870
M120   C11   C5  2777
M121   C11   C6  2509
M122   C11   C7  2390
M123   C11   C9  2241
M124   C11   C10  2490
M125   C11   C13  2924
M126   C12   C1  2675
M127   C12   C2  2031
M128   C12   C4  2157
M129   C12   C5  2514
M130   C12   C7  2753
M131   C12   C9  2561
M132   C12   C10  2257
M133   C12   C11  2620
M134   C12   C13  2411
M135   C13   C0  2469
M136   C13   C1  2697
M137   C13   C2  2454
M138   C13   C3  2193
M139   C13   C4  2709
M140   C13   C6  2629
M141   C13   C7  2694
M142   C13   C9  2853
M143   C13   C12  2219
