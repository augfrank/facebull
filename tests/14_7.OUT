29873
6 16 22 36 43 60 74 81 91 100 115 123 127 138
